feature_annotations.yaml

```diff
@@ -0,0 +1,27 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
```

Comment:
Question (not blocking for this PR): Where do we plan to explain the semantics of each of these fields?  Should the how-to for toggle annotations live with the OEP?

Reply:
From the poll I took on the dev channel, people seem to expect it in the OEP, yes.

Reply:
Perhaps this file should be named "toggle_annotations.yaml"?

Reply:
Yes, or "feature_toggle_annotations.yml". Something with "toggle" in the name.